chat.js

```diff
@@ -1106,13 +1232,16 @@ Chat.plural = function (num, plural = 's', singular = '') {
  */
 Chat.toTimestamp = function (date, options) {
 	const human = options && options.human;
+	/** @type {Array<number>} */
```

Comment:
PS standard is to match TypeScript standard: `number[]` rather than `Array<number>`

Reply:
Also, I'd use `any[]`. Don't worsen optimization just to make TypeScript happy.